enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -180,28 +184,133 @@ func pebbleCacheFromConfig(env environment.Env, cfg *PebbleCacheConfig) (*pebble
 	return c, nil
 }
 
-func (mc *MigrationCache) doubleRead() bool {
-	return mc.doubleReadPercentage > 0 && rand.Float64() < mc.doubleReadPercentage
+// Flags and values for experiments which control the migration cache.
+// Exported for testing.
+const (
+	MigrationCacheConfigFlag      = "migration-cache-config"
+	MigrationStateField           = "state"
+	AsyncDestWriteField           = "async-dest-write"
+	DoubleReadPercentageField     = "double-read-percentage"
+	DecompressReadPercentageField = "decompress-read-percentage"
+
+	SrcOnly     = "src-only"
```

Comment:
Within each state, what's the intention for how fields should be adjusted? Like if you want to ramp up the double read percentage for src-primary, would you just edit that field in the config?